webots_ros2_tests/test/test_system_turtlebot_tutorial_navigation.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+
+# Copyright 1996-2021 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test the `webots_ros2_turtlebot` package on the SLAM and Navigation tutorials."""
+
+# Launch the test locally: launch_test src/webots_ros2/webots_ros2_tests/test/test_system_turtlebot_tutorial_navigation.py
+
+import os
+import pytest
+import rclpy
+from geometry_msgs.msg import PoseWithCovarianceStamped
+from geometry_msgs.msg import Point
+from geometry_msgs.msg import Quaternion
+from nav_msgs.msg import OccupancyGrid
+from launch import LaunchDescription
+import launch_testing.actions
+from ament_index_python.packages import get_package_share_directory
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.actions import IncludeLaunchDescription
+from webots_ros2_tests.utils import TestWebots, initialize_webots_test
+
+turtlebot3_map = 'turtlebot3_burger_example_map.yaml'
+
+
+@pytest.mark.rostest
+def generate_test_description():
+    initialize_webots_test()
+    # If ROS_REPO is testing or ROS_DISTRO is rolling, skip test so CI succeed
+    if ('ROS_REPO' in os.environ and os.environ['ROS_REPO'] == 'testing') or \
+            ('ROS_DISTRO' in os.environ and os.environ['ROS_DISTRO'] == 'rolling'):
```

Comment:
Why do we need to skip rolling, it should work on rolling, isn't it?

Reply:
The packages needed to run this 2 tutorials are not available for `Galactic` and `Rolling`, thus we need to skip the test in case we are using those distributions.

But we should be able to run it with `Foxy testing` now, I will check that.